packages/connect/src/types/index.ts

```diff
@@ -44,3 +44,4 @@ export type {
 } from '@trezor/blockchain-link';
 
 export { FirmwareType, type VersionArray } from '@trezor/device-utils';
+export { ThpPairingMethod } from '@trezor/protocol';
```

Comment:
```suggestion
export { ThpPairingMethod } from '@trezor/protocol/src/types';
```
We need to restrict the import to avoid pulling in the 'crypto' module, which currently breaks the build.